### PR TITLE
fix: switching to @cryptkeeperzk/rlnjs and @cryptkeeperzk/semaphore

### PR DESCRIPTION
**Pull request by 0xisk**
## Explanation

This PR remove the forked dependencies from the monorepo and instead it install forked packages directly from the npm registry. 

- `ffjavascript`: https://github.com/CryptKeeperZK/ffjavascript
   - `@cryptkeeperzk/ffjavascript`: https://www.npmjs.com/package/@cryptkeeperzk/ffjavascript
- `snarkjs`: https://github.com/CryptKeeperZK/snarkjs
   - `@cryptkeeperzk/snarkjs`: https://www.npmjs.com/package/@cryptkeeperzk/snarkjs
- `Semaphore`: https://github.com/CryptKeeperZK/semaphore
   - `@cryptkeeperzk/semaphore-proof`: https://www.npmjs.com/package/@cryptkeeperzk/semaphore-proof
   - `@cryptkeeperzk/semaphore-identity`: https://www.npmjs.com/package/@cryptkeeperzk/semaphore-identity
   - `@cryptkeeperzk/semaphore-group`: https://www.npmjs.com/package/@cryptkeeperzk/semaphore-group
- `rlnjs`:  https://github.com/CryptKeeperZK/rlnjs
   - `@cryptkeeperzk/rlnjs`: https://www.npmjs.com/package/@cryptkeeperzk/rlnjs
- `ejs`: https://github.com/CryptKeeperZK/ejs
   -  `@cryptkeeperzk/ejs`: https://www.npmjs.com/package/@cryptkeeperzk/ejs

## More Information

* Closes https://github.com/CryptKeeperZK/crypt-keeper-extension/issues/667
* Related #685  
* Ref: https://github.com/CryptKeeperZK/crypt-keeper-extension/pull/685#issuecomment-1679384040

## Screenshots/Screencaps

N/A

## Manual Testing Steps

N/A

## Pre-Merge Checklist

- [X] PR template is filled out
- [X] Pre-commit and pre-push hook checks are passed
- [ ] E2E tests are passed locally
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [X] PR is linked to the appropriate GitHub issue
- [X] PR has been added to the appropriate release Milestone

> PR template source from [github.com/MetaMask](https://github.com/MetaMask)

### CODE REVIEW

File: pnpm-workspace.yaml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 packages:
-  - "packages/*"
-  - "packages/proofs/*"
+  - "packages/*"
```
Comment (0xmad) [inline pnpm-workspace.yaml:2]: Please add empty line at the end of this file

**Review by 0xmad — APPROVED** [review]
@0xisk thanks, just one code style comment.

**Comment by 0xmad** [comment]
@0xisk local launch shows that the branch code has some permission issues
image

**Comment by 0xisk** [comment]
@

> @0xisk local launch shows that the branch code has some permission issues image image
> 
> For firefox doesn't launch too. image

Thanks @0xmad for your review, that was because of `ejs.js` package used inside `snarkjs.js`. 
I have fixed this package issue as you could see from here: https://github.com/CryptKeeperZK/ejs
And it is discussed in detail in this comment: https://github.com/CryptKeeperZK/ejs/blob/main/lib/ejs.js#L101-L118

This forked package is published under `@cryptkeeperzk` in the npm registry: https://www.npmjs.com/package/@cryptkeeperzk/ejs

**Comment by 0xmad** [comment]
Run full e2e tests on this branch. When they are passed we can merge it.
https://github.com/CryptKeeperZK/crypt-keeper-extension/actions/runs/5904448304/job/16017405370

**Pull request merged**